straxen/plugins/events/event_se_score.py

```diff
@@ -2,18 +2,18 @@
 import strax
 
 
-class EventSEDensity(strax.Plugin):
+class EventSEScore(strax.Plugin):
     """This plugin is designed to calculate the single electron rate density for main and alt S2 in
     events.
 
     References:
-        * v0.0.0: xenon:xenonnt:analysis:hot_spot_cut_summary
+        * v0.1.0: xenon:xenonnt:ac:sr1:hotspot_veto_cut:wimp_roi
```

Comment:
Reference to v0.0.0 should also be kept.

Reply:
Ok